src/services/practice-programs/PracticePrograms.service.ts

```diff
@@ -18,15 +19,17 @@ export const searchPracticePrograms = async (
 
   if (locationId) requestUrl = `${requestUrl}&locationId=${locationId}`;
 
-  if (faculties) requestUrl = `${requestUrl}&${faculties.map(f => `faculties[]=${f}`).join('&')}`;
+  if (faculties && faculties.length > 0)
+    requestUrl = `${requestUrl}&${faculties.map((f) => `faculties[]=${f}`).join('&')}`;
 
-  if (workingHours) requestUrl = `${requestUrl}&workingHours=${workingHours}`;
+  if (workingHours) requestUrl = `${requestUrl}&workingHours=${workingHours.value}`;
 
-  if (domains) requestUrl = `${requestUrl}&${domains.map(f => `domains[]=${f}`).join('&')}`;
+  if (domains && domains?.length > 0)
+    requestUrl = `${requestUrl}&${domains.map((f) => `domains[]=${f}`).join('&')}`;
```

Comment:
daca punem in `params: {}` e posibil sa mearga direct array

Reply:
Will check.

Reply:
Am un branch in progress unde am sa fac change-urile aici si pentru ong